ui/config/prod.env.js

```diff
@@ -1,4 +1,5 @@
 'use strict'
 module.exports = {
+  TOKBOX_API_KEY: `"${process.env.TOKBOX_API_KEY}"`,
```

Comment:
Shouldn't the backend give it in the response?

Reply:
You are right, fixed in https://github.com/mind-doc/videochat-tutorial/pull/2/commits/7487cd015654cc8c9d6efc10c2fbc59146b42a39